cron/daily/populate_search_index.js

```diff
@@ -0,0 +1,107 @@
+import config from 'config';
+import algoliasearch from 'algoliasearch';
+import Promise from 'bluebird';
+
+import models from '../../server/models';
+import { types as collectiveTypes } from '../../server/constants/collectives';
+import { chunkArray } from '../../server/lib/utils';
+import emailLib from '../../server/lib/email';
+
+const ALGOLIA_APP_ID = config.algolia.appId;
+const ALGOLIA_KEY = config.algolia.appKey;
+const ALGOLIA_INDEX = 'collectives';
+const chunkSize = 10; // number of collectives to send at once
+
+
+const done = (err) => {
+  if (err) {
+    console.log('err', err)
+    return emailLib.sendMessage(
+    'ops@opencollective.com', 
+    'Error in updating index', 
+    '', {
+      bcc: ' ',
+      text: err
+    })
+    .then(process.exit)
+    .catch(console.error)
+  }
+  console.log('Finished updating search records');
+  process.exit();
+}
+
+
+// TODO: clean index once a week
+const populateIndex = () => {
+  return models.Collective.findAll({
```

Comment:
this works, but FYI, you can also use `async` `await` which makes up for a cleaner code imho.